client/web/src/ui/update-available.tsx

```diff
@@ -0,0 +1,53 @@
+import React from "react"
+import { ClientVersion } from "src/hooks/self-update"
+import { Link } from "wouter"
+
+export function UpdateAvailableNotification({
```

Comment:
Since this component is actually tailscale-specific (uses client version, etc), I wouldn't hold it in the UI folder - that's more reserved for general UI components. You could throw this into the updating-view.tsx file and export it, or move it into the /components folder.